controllers/feeds/getFeedsV2.js

```diff
@@ -17,7 +17,7 @@ const {
   modifyAnonimityPost,
   isPostBlocked,
   modifyReactionsPost,
-  modifyFeedIsFollowingTarget
+  isUserFollowAuthor
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `isUserFollowAuthor` is replacing `modifyFeedIsFollowingTarget`. Ensure that the new function `isUserFollowAuthor` has the same functionality as the old one or that the changes are intended and won't break existing functionality.



<!-- This is an auto-generated comment by OSS CodeRabbit -->